buildpipeline/pipeline.json

```diff
@@ -130,6 +130,29 @@
           }
         },
         {
+          "Name": "Core-Setup-CrossBuild",
+          "Parameters": {
+            "PB_Architecture": "arm"
```

Comment:
Yes, but being explicit here can make it more obvious what's being set (it might not be obvious why one job passes values for certain parameter while another doesn't) - we do this for coreFX already (https://github.com/dotnet/corefx/blob/master/buildpipeline/pipeline.json#L159), and in the CoreCLR pipelines.json file in VSO. Also relying on default values can be dangerous, in case they change - although in this case that's unlikely to happen.